paymentlogic.js

```diff
@@ -0,0 +1,40 @@
+
+function basic(salary)
```

Comment:
try to use ES6 standard

ex: const basic = (salary) => ...
use let or const instead of var